graphql_compiler/schema_transformation/rename_query.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from copy import deepcopy
+
+from graphql.language import ast as ast_types
+from graphql.language.visitor import Visitor, visit
+
+from ..exceptions import GraphQLValidationError
+
+
+def rename_query(ast, renamings):
+    """Translate names of types and root vertex fields using renamings.
+
+    Besides root vertex fields (fields of the query type), no other fields will be renamed.
```

Comment:
I don't think this is sufficient -- what about `InlineFragment` AST objects, which we use to perform type coercions?

Reply:
All types will be renamed, and only root vertex fields will be renamed. So for example, in 
```
{
  Character {
    friend {
      ... on Human {
        name
      }
    }
  }
}
```
Character will be renamed since it's a root vertex field, Human will be renamed since it's a type, friend and name will not be renamed since they're normal fields.

Reply:
It might be good to add a sentence into the docstring saying something to this effect. For example:

```
All type names (including ones in type coercions), as well as root vertex fields (fields of the query type) will be renamed. No other field names will be renamed.
```